homework6/infinity_cycle.py

```diff
@@ -0,0 +1,15 @@
+""" Infinity cycle.
+This program identifies, leads the code to an infinite cycle.
+"""
+
+
+def is_infinity_cycle(a, b):
+    """This function identifies an infinite cycle."""
+    while a != b:
+        a += 1
+        b -= 1
+    return a != b
+
+
+print(f"Этот цикл будет бесконечен. {is_infinity_cycle(2, 3)}")
+print(f"Этот цикл НЕ бесконечен. {is_infinity_cycle(2, 6)}")
```

Comment:
if I understood correctly, in case of infinity loop -> program will not stop until OS kill the process?
The idea here is to detect infinity loop and return `true` or `false`, as in examples that I have sent to you